libraries/DateTime/contracts/DateTime.aes

```diff
@@ -114,13 +62,191 @@ contract DateTime =
     let timestamp = timestamp + second  
     timestamp
 
-// Private functions and internal logic
+  public function parse_timestamp(timestamp : int) : date_time = 
+    let dt : date_time = init_date_time(timestamp)
```

Comment:
why use short variable names? as a good example we should use semantic naming for variables

(true for all cases in this contracts)

Reply:
still open

Reply:
as long as we have let dt followed by its type, it is understandable enough i believe.

Reply:
ok